src/groupInfo/components/GroupGalleryUI.vue

```diff
@@ -182,6 +189,7 @@ export default {
   },
   data () {
     return {
+      width: -1,
```

Comment:
Why `-1` as a default?

Reply:
Oh yes, I was going use it somewhere to check if it was uninitialised... I'll have a look again! Not entirely sure how much it's possible to rely on the resize Oberserver to initialize the value...

Reply:
I commented it now, and added in a check for `!== -1` where it gets used.